operators/azure/azure_invokers.hpp

```diff
@@ -1,43 +1,44 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-
 #pragma once
 
 #include "ocos.h"
+#include "curl_invoker.hpp"
 
-struct AzureInvoker : public BaseKernel {
-  AzureInvoker(const OrtApi& api, const OrtKernelInfo& info);
-
- protected:
-  ~AzureInvoker() = default;
-  std::string model_uri_;
-  std::string model_name_;
-  std::string model_ver_;
-  std::string verbose_;
-  std::vector<std::string> input_names_;
-  std::vector<std::string> output_names_;
-};
-
-struct AzureAudioInvoker : public AzureInvoker {
-  AzureAudioInvoker(const OrtApi& api, const OrtKernelInfo& info);
-  void Compute(const ortc::Variadic& inputs, ortc::Tensor<std::string>& output);
+namespace ort_extensions {
 
- private:
-  std::string binary_type_;
-};
+////////////////////// AzureAudioToText //////////////////////
 
-struct AzureTextInvoker : public AzureInvoker {
-  AzureTextInvoker(const OrtApi& api, const OrtKernelInfo& info);
-  void Compute(std::string_view auth, std::string_view input, ortc::Tensor<std::string>& output);
+/// <summary>
+/// Azure Audio to Text
+/// Input: auth_token {string}, ??? (Update when AOAI endpoint is defined)
+/// Output: text {string}
+/// </summary>
+class AzureAudioToText : public CurlInvoker {
```

Comment:
Should we suffix the names with Invoker? Like AzureAudioToTextInvoker or is that too verbose? Rationale is to know instantly that this class is an invoker. For e.g. we've AzureTritonInvoker.

Reply:
I'm fine with adding the suffix to add clarity.

Reply:
I've added 'Invoker' to the class names. I didn't change the operator names at this point as the 'Invoker' part doesn't seem meaningful enough at that level to warrant the longer name.